pkg/sql/logictest/logic.go

```diff
@@ -1287,6 +1300,7 @@ func (t *logicTest) newTestServerCluster(bootstrapBinaryPath string, upgradeBina
 	t.clusterCleanupFuncs = append(t.clusterCleanupFuncs, ts.Stop)
```

Comment:
nit: as long as the cleanupFuncs are being added here, should we just have your new callback get appended here too, rather than returning it. either way, it seems better to be consistent either append both cleanupFunctions here, or to the `ts.Stop` call in the returned function.

Reply:
Right, I don't know what I was thinking. Fixed.